stdlib/std.jsonnet

```diff
@@ -39,7 +39,7 @@ limitations under the License.
     assert std.isString(str) : 'substr first parameter should be a string, got ' + std.type(str);
     assert std.isNumber(from) : 'substr second parameter should be a string, got ' + std.type(from);
     assert std.isNumber(len) : 'substr third parameter should be a string, got ' + std.type(len);
-    assert len >=0 : 'substr third parameter should be greater than zero, got ' + len;
+    assert len >= 0 : 'substr third parameter should be greater than zero, got ' + len;
```

Comment:
These changes are also all autoformatting but they look like good changes? I can rebase and just take the actual stuff I added if you want.

Reply:
Yeah, it looks like our own auto-formatting. Ideally that would be a separate commit, but that's not a big deal.